.github/workflows/pr-title-check.yml

```diff
@@ -0,0 +1,36 @@
+name: Check PR Title
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+jobs:
+  check-pr-title:
+    name: Check PR Title (conventional-commits)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR Title Format
+        env:
+          PR_TITLE: ${{ github.event.pull_request.title }}
+        run: |
+          fail() {
+            echo -e "ERROR: Pull request title does not match the Conventional Commits format.\n"
+            echo -e "Required format: <type>(<scope>): <message> or <type>: <message>\n"
+            echo -e "Valid types: feat, fix, chore, docs, style, refactor, perf, test, build, ci, revert\n"
+            echo -e "Your title: $PR_TITLE\n"
+            echo "See: https://www.conventionalcommits.org/en/v1.0.0/"
+            exit 1
+          }
+          
+          echo "Checking PR title: $PR_TITLE"
+          
+          PATTERN='^[a-z]+(\([^)]+\))?(!)?: .+'
+          VALID_TYPES="feat|fix|chore|docs|style|refactor|perf|test|build|ci|revert"
+          
+          [[ ! "$PR_TITLE" =~ $PATTERN ]] && fail
```

Comment:
The workflow validates PR titles in two steps:
1. Check the format (type, scope, colon, message)